recipes-graphics/vulkan/vulkan-loader_%.bbappend

```diff
@@ -1,3 +1,5 @@
 # The i.MX implementation is dynamically loaded, so it requires an
 # explicit runtime dependency.
-RRECOMMENDS:${PN}:append:imxgpu = " libvulkan-imx"
+PACKAGES_VULKAN:aarch64       = "libvulkan-imx"
+PACKAGES_VULKAN:mx8mm-nxp-bsp = ""
+RRECOMMENDS:${PN}:append:imxgpu = " ${PACKAGES_VULKAN}"
```

Comment:
```
PACKAGES_VULKAN:imxgpu        = " libvulkan-imx"
PACKAGES_VULKAN:mx8mm-nxp-bsp = ""
RRECOMMENDS:${PN}:append      = "${PACKAGES_VULKAN}"
```

Reply:
look better! test local and works

Reply:
We do only support `aarch64`, so the original was correct.

Actually, I was working on this as well and have this:

```
RRECOMMENDS:${PN} += "${RRECOMMENDS_IMX}"

RRECOMMENDS_IMX                = ""
RRECOMMENDS_IMX:aarch64        = "${RRECOMMENDS_IMXGPU}"
RRECOMMENDS_IMX:mx8mm-nxp-bsp  = ""

RRECOMMENDS_IMXGPU             = ""
RRECOMMENDS_IMXGPU:imxgpu      = "libvulkan-imx"
```

Reply:
It is not, as otherwise non-imx machines will end appending to it.

Reply:
I use the same variable PACKAGES_VULKAN to make it more consistent with the change https://github.com/Freescale/meta-freescale/commit/ee92b5a8b2cad45de267d3cf6dae9251f5ed11fd and as it is a consequence of this change.

@thochstein your solution can add the possibility to additionally select the vulkan lib so I will use your suggestion

Reply:
I wonder if we shouldn't have an imxvulkan so we abstract it in imx-base.

Reply:
Yes, a new `imxvulkan` in `MACHINEOVERRIDES_EXTENDER` can work as well.

Reply:
What should I do, I leave it as it is or create a new `imxvulkan` override?

Reply:
Let's create the new override, it makes it easier to handle.

Reply:
Thanks, I will do that

Reply:
Don't know if this the way of creating an override in this project but here it is, any suggestion is welcome